mne/beamformer/resolution_matrix.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Compute resolution matrix for beamformers."""
+# Authors: olaf.hauk@mrc-cbu.cam.ac.uk
+#
+# License: BSD (3-clause)
+import numpy as np
+
+from ..io.pick import pick_channels, pick_info, pick_channels_forward
+from ..evoked import EvokedArray
+from ..utils import logger
+from ._lcmv import apply_lcmv
+
+
+def make_lcmv_resolution_matrix(filters, forward, info):
+    """Compute resolution matrix for LCMV beamformer.
+
+    Parameters
+    ----------
+    filters : instance of Beamformer
+         Dictionary containing filter weights from LCMV beamformer
+         (see mne.beamformer.make_lcmv).
+    forward : dict
+        Forward Solution with leadfield matrix.
+    info : instance of Info
+        Measurement info used to compute LCMV filters.
+
+    Returns
+    -------
+    resmat : array, shape (n_dipoles_lcmv, n_dipoles_fwd)
+        Resolution matrix (filter matrix multiplied to leadfield from
+        forward solution). Numbers of rows (n_dipoles_lcmv) and columns
+        (n_dipoles_fwd) may differ depending on orientation constraints of
+        filter and forward solution, respectively.
```

Comment:
can you be more explicit? Saying that n_dipoles_lcmv can be 3 times lower than n_dipoles_fwd if a scalar beamformer is used.